test/e2e/admission_controller.go

```diff
@@ -0,0 +1,143 @@
+/*
+Copyright 2015 The Kubernetes Authors.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+/**
+this component is purposed to tests webhooks in the admission controller
+*/
+
+package e2e
+
+import (
+	"fmt"
+	"strings"
+	"time"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	v1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/util/uuid"
+
+	appsv1 "k8s.io/api/apps/v1"
+	"k8s.io/api/extensions/v1beta1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/labels"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/kubernetes/test/e2e/framework"
+)
+
+const (
+	deploymentId = "d-2ajbkvmtqo5isznh3m2raj2bdp"
+	pipelineId   = "l-2kwqgqevuqwsje5a9ukhsk4bdd"
+)
+
+var _ = framework.KubeDescribe("Admission controller tests", func() {
+	f := framework.NewDefaultFramework("zalando-kube-admission-controller")
+	var cs kubernetes.Interface
+
+	BeforeEach(func() {
+		cs = f.ClientSet
+	})
+
+	It("Pods should get deployment info from deployment and zone from node [Zalando]", func() {
```

Comment:
I'd add a test that ensures that the users can override the variable and we wouldn't overwrite it with the default values.